esp-config/src/generate.rs

```diff
@@ -0,0 +1,138 @@
+use core::fmt::Write;
+use std::{collections::HashMap, env, fs, path::PathBuf};
+
+const TABLE_HEADER: &str = r#"
+| name | description | default value |
```

Comment:
```suggestion
| Name | Description | Default value |
```

Upper Case Headings Rule